test/e2e/framework/util.go

```diff
@@ -3068,7 +3071,7 @@ func DeleteResourceAndWaitForGC(c clientset.Interface, kind schema.GroupKind, ns
 		return err
 	}
 
-	ps, err := podStoreForSelector(c, ns, selector)
+	ps, err := testutils.NewPodStore(c, ns, selector, fields.Everything())
```

Comment:
Is that actually equivalent to the previous version? In old code it's passing what looks like a specific selector and here you just pass fields.Everything?

Reply:
Ok, please ignore, just saw the method below.